profitbricks/src/main/java/org/jclouds/profitbricks/config/ProfitBricksComputeServiceContextModule.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.profitbricks.config;
+
+import org.jclouds.compute.ComputeServiceAdapter;
+import org.jclouds.compute.config.ComputeServiceAdapterContextModule;
+import org.jclouds.compute.domain.Hardware;
+import org.jclouds.compute.domain.NodeMetadata;
+import org.jclouds.compute.domain.Volume;
+import org.jclouds.domain.Location;
+import org.jclouds.functions.IdentityFunction;
+import org.jclouds.profitbricks.compute.ProfitBricksComputeServiceAdapter;
+import org.jclouds.profitbricks.domain.DataCenter;
+import org.jclouds.profitbricks.domain.Image;
+import org.jclouds.profitbricks.domain.Server;
+import org.jclouds.profitbricks.domain.Storage;
+import org.jclouds.profitbricks.compute.function.DataCenterToLocation;
+import org.jclouds.profitbricks.compute.function.ImageToImage;
+import org.jclouds.profitbricks.compute.function.ServerToNodeMetadata;
+import org.jclouds.profitbricks.compute.function.StorageToVolume;
+
+import com.google.common.base.Function;
+import com.google.inject.TypeLiteral;
+
+public class ProfitBricksComputeServiceContextModule extends
+	ComputeServiceAdapterContextModule<Server, Hardware, Image, DataCenter> {
+
+   @Override
+   protected void configure() {
+      super.configure();
+
+      // FIXME Not working
+      install(new LocationsFromComputeServiceAdapterModule<Server, Hardware, Image, DataCenter>() {
```

Comment:
That's because it is qualified when bound in the module that configures it:
https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/location/config/LocationModule.java#L102-L106